app/grandchallenge/challenges/emails.py

```diff
@@ -91,3 +91,24 @@ def send_challenge_status_update_email(challengerequest, challenge=None):
         recipients=[challengerequest.creator],
         subscription_type=EmailSubscriptionTypes.SYSTEM,
     )
+
+
+def send_email_percent_budget_consumed_alert(challenge, warning_threshold):
+    send_standard_email_batch(
+        site=Site.objects.get_current(),
+        subject=format_html(
+            "[{challenge_name}] Challenge {warning_threshold}% Budget Consumed Alert",
+            challenge_name=challenge.short_name,
+            warning_threshold=warning_threshold,
+        ),
+        markdown_message=format_html(
+            "We would like to inform you that {percent_budget_consumed}% of the "
+            "compute budget for your challenge has been used.",
+            percent_budget_consumed=challenge.percent_budget_consumed,
+        ),
+        recipients=[
+            *challenge.get_admins(),
```

Comment:
Initially I had the `challenge.creator` here explicitly. But I think they are always in the admins group? If I include the creator here separately I end up sending the email to them twice.

Reply:
The creator is just for tracking so should be treated as anything special. Only the admins group should be emailed, think of the following situations:

- We create a challenge on behalf of another user, we are then removed from the admins group.
- Someone creates a challenge, later leaves the organisation and is removed from the admins group by the other admins.

In both situations we do not want the creator to get an email. They will get an email if they are part of the admins group, which is what we want.

Reply:
Sending this to the admins group is fine, I think. The creator is indeed automatically added to the admins and will stay an admin unless another admin removes him/her, in which case you wouldn't want to email them.